ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos-intended-config.j2

```diff
@@ -98,6 +98,8 @@
 {% include 'eos/enable-password.j2' %}
 {# aaa root #}
 {% include 'eos/aaa-root.j2' %}
+{# CVX #}
```

Comment:
please move the include to the correct position according to the order of configuration rendered by EOS.

Reply:
I think it is in the right spot.
![Screen Shot 2022-08-16 at 11 46 22 AM](https://user-images.githubusercontent.com/4846759/184955535-2c69ba1b-288c-48b9-8f11-174e148de63a.png)

Reply:
I meant the _order_ of configuration. So you have it right below aaa-root which is not where you find it in "show run". On my test dut it came right above the ethernet interfaces, but I think that will depend on what other config is there.

Reply:
I tested some more and you should move cvx below "link-tracking-groups.j2" and above "port-channel-interfaces.j2"